src/scenes/SearchResults/Search.graphql

```diff
@@ -11,9 +11,47 @@ fragment SearchResultItem on SearchResult {
   ...LanguageListItem
   ...OrganizationListItem
   ...UserListItem
-  ...DisplayLocation
+  ...LocationSearchResultItem
   ...FilmLookupItem
   ...StoryLookupItem
   ...LiteracyMaterialLookupItem
   ...SongLookupItem
 }
+
+# We have to inline this because the fragment we would usually
+# insert here has a `type` field that conflicts with a field of the
+# same name on the `Project` type, so we need to use an alias.
```

Comment:
Is this a TS error or gql gen error?

Reply:
gql-gen error

Reply:
Wait I don't think we actually need type in the search results do we? If we are showing them on the card I guess we do.
I'm ok leaving this for now then as long as gql gen gets that rename into the TS types. It does right?

Reply:
Yeppers